lib/model.js

```diff
@@ -2110,8 +2110,15 @@ Model.findById = function findById(id, projection, options) {
  *     // Find one adventure whose `country` is 'Croatia', otherwise `null`
  *     await Adventure.findOne({ country: 'Croatia' }).exec();
  *
- *     // using callback
+ *     // Model.findOne() no longer accepts a callback
```

Comment:
Let's just get rid of the callback section, L2113-L2121. I think `await Adventure.findOne({ country: 'Croatia' }).exec()` is sufficient.

Reply:
Okay done.